app/service/rest.py

```diff
@@ -345,7 +345,11 @@ def create_api_key(service_id=None):
     valid_api_key.service = fetched_service
     save_model_api_key(valid_api_key)
     unsigned_api_key = get_unsigned_secret(valid_api_key.id)
-    return jsonify(data=unsigned_api_key), 201
+
```

Comment:
This is where API stamps the prefix on and passes it back to the user (via admin). The data contract had to change here, so admin requires an update for this as well.